estate_account/__manifest__.py

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Real Estate Accounting",
+    "description": "Real Estate Accounting (link module)",
+    "author": "robal",
```

Comment:
```suggestion
    "author": "Odoo S.A.",
```
As an employee, you work in the name of Odoo.